File: testprojects/master_scala_mainone/pipelines/SCALA_LIVY/code/src/main/scala/io/prophecy/pipelines/scala_livy/graph/Reformat_1.scala
package io.prophecy.pipelines.scala_livy.graph

import io.prophecy.libs._
import io.prophecy.pipelines.scala_livy.config.ConfigStore._
import io.prophecy.pipelines.scala_livy.config.Context
import io.prophecy.pipelines.scala_livy.udfs.UDFs._
import io.prophecy.pipelines.scala_livy.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._
object Reformat_1 { def apply(context: Context, in: DataFrame): DataFrame = in }

File: testprojects/python_main/pipelines/PYTHON_AZURE/code/python_azure/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from python_azure.config.ConfigStore import *
from python_azure.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0
